tests/common.py

```diff
@@ -1217,7 +1226,7 @@ def mock_storage(data=None):
 
     orig_load = storage.Store._async_load
 
-    async def mock_async_load(store):
+    async def mock_async_load(store: storage.Store) -> None:
```

Comment:
This will return the storage generic type of the stored data.

https://github.com/home-assistant/core/blob/9404a107b2c052660ce40bced894ec515b238938/homeassistant/helpers/storage.py#L119

Reply:
Ah yes! That was part of the reason why I didn't type it originally.
Since we don't know _T here, maybe it should be:
```python
    async def mock_async_load(store: storage.Store) -> dict[str, Any] | list[Any] | None:
```